server_config.go

```diff
@@ -75,6 +75,7 @@ func (cfg *ServerConfig) makeCommand() func() *exec.Cmd {
 	var cmd = strings.Split(cfg.Command, " ")
 	return func() *exec.Cmd {
 		cmd := exec.Command(cmd[0], cmd[1:]...)
+		cmd.Env = append(os.Environ(), fmt.Sprintf("RR_RELAY=%s", cfg.Relay))
 		cmd.Env = append(os.Environ(), cfg.env...)
```

Comment:
`cmd.Env = append(cmd.Env, cfg.env...)`?

Reply:
yep, added fix for that + additional tests. Thx for catching.

Reply:
This value won't be referenced in Wiki (https://github.com/spiral/roadrunner/wiki/Enviroment-Configuration) until next release.